src/test/resources/com/puppycrawl/tools/checkstyle/api/filetext/InputFileTextImportControl.xml

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0"?>
 <!DOCTYPE import-control PUBLIC
-    "-//Checkstyle//DTD ImportControl Configuration 1.0//EN"
-    "https://checkstyle.org/dtds/import_control_1_0.dtd">
+    "-//Checkstyle//DTD ImportControl Configuration 1.5//EN"
+    "https://checkstyle.org/dtds/import_control_1_5.dtd">
```

Comment:
lets not touch old test that use old dtd, it helps proving backward compatibility.
in this PR, new test is expected only.